examples/analytics.amp.html

```diff
@@ -99,52 +169,72 @@
       }
     }
   }
-</script>
-</amp-analytics>
-<!-- End AT Internet tracking -->
+</script> </amp-analytics>
+		<!-- End AT Internet tracking -->
 
-<!-- Burt tracking -->
-<amp-analytics type="burt">
-  <script type="application/json">
+		<!-- Burt tracking -->
+		<amp-analytics type="burt"> <script
+			type="application/json">
     {
       "vars": {
         "trackingKey": "B31VT4QH1BFR",
         "category": "sports",
         "subCategory": "golf"
       }
     }
-  </script>
-</amp-analytics>
-<!-- End Burt example -->
+  </script> </amp-analytics>
+		<!-- End Burt example -->
 
-<!-- Chartbeat tracking -->
-<amp-analytics type="chartbeat">
-<script type="application/json">
+		<!-- Chartbeat tracking -->
+		<amp-analytics type="chartbeat"> <script
+			type="application/json">
 {
   "vars": {
     "uid": "12345",
     "domain": "my-site.com",
     "sections": "section 1, section 2"
   }
 }
-</script>
-</amp-analytics>
-<!-- End Chartbeat example -->
+</script> </amp-analytics>
+		<!-- End Chartbeat example -->
 
-<!-- comScore UDM pageview tracking -->
-<amp-analytics type="comscore">
-<script type="application/json">
+		<!-- ColAnalytics pageview tracking -->
+		<amp-analytics type="colanalytics"> <script
+			type="application/json">
+{
+    "requests": {
+        "host": "https://${colurl}",
```

Comment:
colurl is not defined in the example. So when people load this page, they are going to see errors on the page. Can you use a valid example so that the tag correctly sends data to the server?